<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrugatorAnalysis</class>
 <widget class="QMainWindow" name="CorrugatorAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Corr. Analysis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../universal/Leva/src/res.qrc">
    <normaloff>:/images/Icon_12.ico</normaloff>:/images/Icon_12.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select the production date range:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpStartDate">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpEndDate">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdRecalculate">
        <property name="text">
         <string>Recalculate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdView">
        <property name="text">
         <string>View Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="txtAnalysis"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../universal/Leva/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
